unc-csxl/csxl.unc.edu
# Task/CSXL-597: Recurring Office Hours Events

This PR adds functionality for recurring OH events on the backend and frontend. Specifically, it allows for the creation and deletion of recurring events.

## Testing
1. Click on the button to create a new event for a course site.
2. Set the recurring events option to "weekly".
3. Select which days of the week the event should recur on.
4. Select an end date for the recurrence.
5. Ensure that all expected events are created (either in the current/future section).

## Future Development
- When deleting a recurring event, create a UI to show the next few events in the pattern that will be deleted.
- Add update functionality for recurring events.
- Restrict end date to end of term / two weeks after EOT.

> _This PR resolves #597._
